pipeline-modules/app-service/azure/node-app-build.yaml

```diff
@@ -0,0 +1,72 @@
+provisioner: azure
+name: node-app-build
```

Comment:
https://github.com/cldcvr/codepipes-ci-cd-modules/blob/master/pipeline-modules/app-service/gcp/node-build.yaml

1. GCP already have a module called `node-build` we should follow the same convention here?
2. "Archive files" should be part of the publish-archive module so that node-build may be used independently when we don't want to push archive along with it.

Reply:
done